tests/detect-ftp/ftp-completion-code-01/input.rules

```diff
@@ -0,0 +1,11 @@
+alert ftp any any -> any any (msg: "Match on FTP completion-code #1"; flow:to_client; ftp.completion_code; content:"331"; sid:1;)
```

Comment:
Do we have to specify `flow:to_client; ` ? Is this not implied by `ftp.completion_code` ?

Reply:
`established` is sufficient.